jenkins/ci.suse.de/pipelines/openstack-maintenance-gating.Jenkinsfile

```diff
@@ -13,7 +13,7 @@ pipeline {
 
   agent {
     node {
-      label 'cloud-trigger'
+      label 'cloud-ci-trigger'
```

Comment:
Did I miss this in #3516 ?

Reply:
no, you didn't. This PR was opened before https://github.com/SUSE-Cloud/automation/pull/3516 was merged